swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -148,6 +148,11 @@ private module Cached {
     // flow through `!`
     nodeFrom.asExpr() = nodeTo.asExpr().(ForceValueExpr).getSubExpr()
     or
+    // flow through `?`
+    nodeFrom.asExpr() = nodeTo.asExpr().(BindOptionalExpr).getSubExpr()
+    or
+    nodeFrom.asExpr() = nodeTo.asExpr().(OptionalEvaluationExpr).getSubExpr()
+    or
```

Comment:
Sorry for the wait @geoffw0, I intended to quickly add tests for this but fell into a rabbit hole where a MaD row I needed to add so that the optional access could be tested didn't work as expected. To not waste more time, I've added the test cases with `MISSING` tags — we should be able to remove those once we fix the root cause of the MaD row not working.